istio-control/istio-discovery/templates/telemetryv2_1.5.yaml

```diff
@@ -161,20 +173,23 @@ spec:
         operation: INSERT_BEFORE
         value:
           name: envoy.filters.http.wasm
-          config:
-            config:
-              root_id: stackdriver_outbound
-              configuration: |
-                {{- if not .Values.telemetry.v2.stackdriver.configOverride }}
-                {"enable_mesh_edges_reporting": {{ .Values.telemetry.v2.stackdriver.topology }}, "disable_server_access_logging": {{ not .Values.telemetry.v2.stackdriver.logging }}, "meshEdgesReportingDuration": "600s"}
-                {{- else }}
-                {{ toJson .Values.telemetry.v2.stackdriver.configOverride | indent 8 }}
-                {{- end }}
-              vm_config:
-                vm_id: stackdriver_outbound
-                runtime: envoy.wasm.runtime.null
-                code:
-                  local: { inline_string: envoy.wasm.null.stackdriver }
+          typed_config:
+            "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+            type_url: type.googleapis.com/envoy.config.filter.http.wasm.v2.Wasm
+            value:
+              config:
+                root_id: stackdriver_outbound
+                configuration: |
+                  {{- if not .Values.telemetry.v2.stackdriver.configOverride }}
+                  {"enable_mesh_edges_reporting": {{ .Values.telemetry.v2.stackdriver.topology }}, "disable_server_access_logging": {{ not .Values.telemetry.v2.stackdriver.logging }}, "meshEdgesReportingDuration": "600s"}
+                  {{- else }}
+                  {{ toJson .Values.telemetry.v2.stackdriver.configOverride | indent 8 }}
```

Comment:
qq: why is this indent 8 and the other one 16? should I bump both by two spaces?

Reply:
Yes. Though best to render the config with overrides set. I am not sure if there is a test for it.

Reply:
Yes meaning update all to 18?